canonical-web-and-design/maas-ui
# Sync maas-ui legacy with launchpad master as of 01/11/2019.

## Done
- Backported changes from launchpad master as of commit `7a29be349`

## QA
- Check that the dashboard has a configuration tab where you can switch active discovery on and off
- Check that you can filter machines my NUMA nodes and SR-IOV support
- Check that navigating between machine details tabs does not refresh the whole page
- Check that machine details network + storage tabs have NUMA node info
- Check that the machine details network tab shows (read-only) member interfaces of bonds and bridges
- Check that the grouping in the subnets page looks good (i.e. not every row has a full length border)
